ci-operator/step-registry/gather/loki/gather-loki-commands.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+function queue() {
+  local TARGET="${1}"
+  shift
+  local LIVE
+  LIVE="$(jobs | wc -l)"
+  while [[ "${LIVE}" -ge 50 ]]; do
+    sleep 1
+    LIVE="$(jobs | wc -l)"
+  done
+  echo "${@}"
+  timeout 1m "${@}" | cut -d' ' -f6- | tac >"${TARGET}" &
+}
+
+if test ! -f "${KUBECONFIG}"
+then
+	echo "No kubeconfig, so no point in gathering loki logs."
+	exit 0
+fi
+
+mkdir -p /tmp/loki-logs
+mkdir -p /tmp/loki/bin
+mkdir -p $ARTIFACT_DIR/loki-logs
+
+curl -o /tmp/loki/bin/logcli-linux-amd64.zip -L "https://github.com/grafana/loki/releases/download/v1.3.0/logcli-linux-amd64.zip"
```

Comment:
We should prefer to build an image with the runtime dependencies for this step

Reply:
logcli removed in favor of curl